src/core/tests/file_util.cpp

```diff
@@ -167,3 +167,229 @@ TEST_F(TrimFileTest, relatice_path_to_source_forward_slash_always_supported) {
     auto str_ptr = ov::util::trim_file_name(file_path.c_str());
     EXPECT_EQ(exp_path, str_ptr);
 }
+
```

Comment:
`openvino/util/file_path.hpp` should be included.

Reply:
Done